src/cli/cli.go

```diff
@@ -9,23 +9,59 @@ import (
 	"github.com/nearform/k8s-kurated-addons-cli/src/services/project"
 
 	"github.com/nearform/k8s-kurated-addons-cli/src/utils/defaults"
+	"github.com/nearform/k8s-kurated-addons-cli/src/services/docker"
+	"github.com/nearform/k8s-kurated-addons-cli/src/utils/logger"
 
 	"github.com/urfave/cli/v2"
 )
 
 type CLI struct {
-	Resources embed.FS
-	CWD       string
+	Resources       embed.FS
+	CWD             string
+	DockerService   docker.DockerService
+	project         project.Project
+	dockerImage     docker.DockerImage
 }
 
-func (c CLI) newProject(cCtx *cli.Context) project.Project {
-	return project.New(
-		cCtx.String("app-name"),
-		cCtx.String("project-directory"),
+func (c *CLI) newProject(cCtx *cli.Context) project.Project {
```

Comment:
We might want to change the name of this method since is not just instantiating a new project but also a new DockerService.

TBH I would split the two things and not couple the creation of a `Project` and the creation of a `DockerService` together.